shared/javascripts/options.js

```diff
@@ -63,6 +63,16 @@ if (Privly === undefined) {
   }
   Privly.options = {};
 
+  // CommonJS Module
+  if (typeof module !== "undefined" && module.exports) {
+    module.exports.options = Privly.options;
+    // load dependencies
+    ls = require("./local_storage.js").ls;
+    Privly.storage = require("./storage.js").storage;
```

Comment:
This is when the script will be loaded as a CommonJS module in the background script. The background script will include a `require("chrome/content/privly-applications/shared/javascripts/options.js")` statement. There isn't any HTML page here.